StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkLogin/LinkLoginViewController.swift

```diff
@@ -210,7 +211,21 @@ final class LinkLoginViewController: UIViewController {
             case .success(let response):
                 self.delegate?.linkLoginViewController(self, signedUpAttachedAndSynchronized: response)
             case .failure(let error):
-                self.delegate?.linkLoginViewController(self, didReceiveTerminalError: error)
+                func completeWithTerminalError() {
+                    self.delegate?.linkLoginViewController(self, didReceiveTerminalError: error)
+                }
+
+                guard let stripeError = error as? StripeError, case .apiError(let stripeAPIError) = stripeError else {
+                    return completeWithTerminalError()
+                }
+
+                // Hack to determine if the error is related to the phone number.
+                // If so, show a generic error message under the phone textfield.
+                if let errorMessage = stripeAPIError.message, errorMessage.contains("phone") {
```

Comment:
I wasn't sure if I should search for the localized string "phone" here, or hardcode the english word. Are remote error messages localized?